app/3.0/toolbox/app-storage.md

```diff
@@ -4,16 +4,12 @@ title: App storage
 
 <!-- toc -->
 
-<div>
-{% include 'outdated.html' %}
-</div>
-
 The app storage family of elements gives you a new set of tools for managing data in your app. The
 initial set includes ready-made elements for integrating with Firebase and PouchDB.
 
 ## Firebase integration
 
-The Firebase 3.0.0 SDK supports a new set of Firebase elements, built with app storage, called
```

Comment:
I think we're going to need to remove this section (or add a note that the PolymerFire elements haven't been updated to 3.0 yet).

Reply:
Talked to @e111077, yes, let's cut this section for now. These elements are outside our org so we'll need to coordinate.